packages/ra-core/src/routing/useRedirect.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import { Link, Routes, Route, useLocation } from 'react-router-dom';
+
+import { FakeBrowserDecorator } from '../storybook//FakeBrowser';
+import { useRedirect as useRedirectRA } from './useRedirect';
+
+export default {
+    title: 'ra-core/routing',
+    decorators: [FakeBrowserDecorator],
+};
+
+const Home = () => {
+    const redirect = useRedirectRA();
+    return (
+        <>
+            <h1>Home</h1>
+            <ul>
+                <li>
+                    <button onClick={() => redirect('/dashboard')}>
+                        Dashboard
```

Comment:
I'd use types as labels for your buttons: Relative url, View name, View name with details, Location, Function

Reply:
Nice idea, applied